packages/server-extension/src/index.ts

```diff
@@ -187,6 +193,177 @@ const contentsRoutesPlugin: JupyterLiteServerPlugin<void> = {
   },
 };
 
+/**
+ * A plugin installing the service worker for and handles communication with the Emscpriten file system.
+ */
+const serviceWorkerPlugin: JupyterLiteServerPlugin<void> = {
+  id: '@jupyterlite/server-extension:service-worker',
+  autoStart: true,
+  requires: [],
+  activate: (app: JupyterLiteServer) => {
+    navigator.serviceWorker
```

Comment:
Probably we could handle `navigator.serviceWorker` not being available, which seems to be the case in a Firefox private window.

So the plugin does not fail at startup.